apps/experiments/models.py

```diff
@@ -322,6 +322,9 @@ def get_invite_url(self) -> str:
             )
         )
 
+    def get_platform_name(self) -> str:
+        return self.channel_session.experiment_channel.platform_display
```

Comment:
You should be able to do
```python
return self.channel_session.experiment_channel.get_platform_display()
```
then you can remove this:
```python
    @property
    def platform_display(self):
        return ExperimentChannel.get_platform_display(self)
```

Reply:
Ah, you're right. Missed that option.